src/common/scripts/services.js

```diff
@@ -0,0 +1,31 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('user_management.common');
+
+    module.service('catchErrors', [
+        function () {
+            this.all = function (response, fields) {
```

Comment:
@maxpeterson I could probably do that, but it must be an open source module if we want to keep this one open as well.